python/lsst/drp/tasks/gbdesAstrometricFit.py

```diff
@@ -279,6 +279,12 @@ class GbdesAstrometricFitConnections(
         deferLoad=True,
         multiple=True,
     )
+    colorCatalog = pipeBase.connectionTypes.Input(
+        doc="The catalog of magnitudes to match to input sources.",
+        name="fgcm_Cycle4_StandardStars",
```

Comment:
Yes, but DM-45640 is going to make a more user-facing version of the catalog that will presumably be constructed from the final cycle by default (it's mentioned in a TODO later), so that should solve this issue.